src/test/java/net/minecraftforge/fml/test/BetterModDependenciesTest.java

```diff
@@ -0,0 +1,55 @@
+package net.minecraftforge.fml.test;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Test;
+
+import net.minecraftforge.fml.common.Loader;
+import net.minecraftforge.fml.common.LoaderException;
+import net.minecraftforge.fml.common.versioning.ArtifactVersion;
+
+public class BetterModDependenciesTest
```

Comment:
Well, PR is named better mod depenendecies, so i reused it. But whatever, i can change.
It's just pure naming it this point